internal/pkg/manifest/env.go

```diff
@@ -214,8 +214,9 @@ type EnvironmentCDNConfig struct {
 
 // AdvancedCDNConfig represents an advanced configuration for a Content Delivery Network.
 type AdvancedCDNConfig struct {
-	Certificate  *string `yaml:"certificate,omitempty"`
-	TerminateTLS *bool   `yaml:"terminate_tls,omitempty"`
+	Certificate  *string         `yaml:"certificate,omitempty"`
+	TerminateTLS *bool           `yaml:"terminate_tls,omitempty"`
+	Static       CDNStaticConfig `yaml:"static,omitempty"`
```

Comment:
What do you think of the following for the key name?
```suggestion
	Static       CDNStaticConfig `yaml:"static_website,omitempty"`
```
It's slightly more verbose but I think more explicit 🤔

Reply:
It's actually where their static assets go. I wonder if `static_website` can correctly convey the meaning 🤔

Reply:
That totally makes sense! How about `static_content` or `static_assets` then?